project/Http4sPlugin.scala

```diff
@@ -292,7 +292,7 @@ object Http4sPlugin extends AutoPlugin {
   lazy val jnrUnixSocket = "com.github.jnr" % "jnr-unixsocket" % V.jnrUnixSocket
   lazy val keypool = Def.setting("org.typelevel" %%% "keypool" % V.keypool)
   lazy val literally = Def.setting("org.typelevel" %%% "literally" % V.literally)
-  lazy val log4catsCore = "org.typelevel" %% "log4cats-core" % V.log4cats
+  lazy val log4catsCore = Def.setting("org.typelevel" %% "log4cats-core" % V.log4cats)
```

Comment:
```suggestion
  lazy val log4catsCore = Def.setting("org.typelevel" %%% "log4cats-core" % V.log4cats)
```

Reply:
wtf. The dep was gone :joy: how is this still here